blog/2022-03-21-vxt-to-join-marketplace.md

```diff
@@ -0,0 +1,40 @@
+---
+slug: vxt-joins-marketplace
+title: "Vxt is on a mission to eradicate boring tasks. Now they’re joining Working Group Two’s product marketplace."
+date: 2022-03-21
+tags: [marketplace, voicemail, partners]
+authors: [marius-waldum]
+---
+
+**Vxt**, a technology communications company in New Zealand, will be adding their voicemail product to Working Group Two’s marketplace. Their aim is to increase visibility and distribution with mobile operators on Working Group Two’s platform. 
+
+<!--truncate-->
+
+The passionate [Vxt](www.vxt.co.nz) Vxt team is on a mission to leverage communications data to automate the most boring work in the world, namely administrative tasks. They’ve made voicemail both simpler and smarter by offering instant transcriptions, smart greetings, email integration, and the ability to access your messages from anywhere. 
```

Comment:
You need to format the link as `[Vxt](https://www.vxt.co.nz/)`  for it to work :)

Reply:
Basic error! :) Thanks!